Editor/Modules/ProfilingModule.cs

```diff
@@ -0,0 +1,21 @@
+using System.Collections.Generic;
+using Unity.ProjectAuditor.Editor.Core;
+using Unity.ProjectAuditor.Editor.Profiling;
+
+namespace Unity.ProjectAuditor.Editor.Modules
+{
+    public class ProfilingModule : ProjectAuditorModule
```

Comment:
Do we actually need this module?

Reply:
Right, I put it here for now because the module basically seemed more "modular".

This merely holds the class instance of the ProfilerAnalyzer for now, so it could also live in the ProjectAnalyzer as a field that the GA code has access to.